workflow-templates/lint-eslint.yml

```diff
@@ -20,9 +20,8 @@ concurrency:
 jobs:
   changes:
     runs-on: ubuntu-latest-low
-    permissions:
-      contents: read
-      pull-requests: read
+    # Required for dorny/paths-filter
+    permissions: read-all
```

Comment:
zizmor is complaining and I'm confused:
```
warning[excessive-permissions]: overly broad permissions
  --> workflow-templates/phpunit-pgsql.yml:36:3
   |
36 | /   changes:
37 | |     runs-on: ubuntu-latest-low
38 | |     # Required for dorny/paths-filter
39 | |     permissions: read-all
   | |     --------------------- uses read-all permissions
40 | |
...  |
59 | |               - 'composer.json'
60 | |               - 'composer.lock'
   | |_______________________________- this job
   |
   = note: audit confidence → High
```

Which part is it trying to read that it can not read otherwise?

Reply:
Honestly no clue, couldn't figure out the reason.
But it seems that it fails because not enough permisisons on private repos ? "Resource not accessible by integration"

I checked here,but didn't find anything useful https://github.com/dorny/paths-filter/issues/123
So I tried this and it seems to work https://github.com/nextcloud-gmbh/server/pull/836

Reply:
Yeah, maybe :thinking: 

Let's keep this closed for now!